src/products/src/products-service/data/products.yaml

```diff
@@ -430,4 +430,262 @@
   style: fishing
   description: Every tackle box needs a silver and red reflective fishing lure.
   price: 9.99
-  image: "12.jpg"
\ No newline at end of file
+  image: "12.jpg"
+- id: 1000567890
+  name: Eyeshadow Palette - Set of 3 Palettes
+  category: beauty
+  style: makeup
+  description: Perfect for trialling different shades, each palette contains 4 different shades.
+  price: 145.00
```

Comment:
What is the reasoning behind the item IDs?
I see some really large IDS (1000567891) and some that are close to the existing items id (101,102...)

Reply:
I've replied to a similar question [here](https://github.com/aws-samples/retail-demo-store/pull/96#discussion_r510249803). 

We are happy to standardise for consistency if required